CHANGELOG.md

```diff
@@ -1,5 +1,20 @@
 # Changelog
 
+## [1.2.16](https://github.com/ably/ably-flutter/tree/v1.2.16)
+
+[Full Changelog](https://github.com/ably/ably-flutter/compare/v1.2.15...v1.2.16)
+
+**Fixed bugs:**
+
+- authHeaders type conversion on Android Plugin is incorrect [\#444](https://github.com/ably/ably-flutter/issues/444)
```

Comment:
We should mention here the new available auth methods

Reply:
9e7b3a8531075050a9ef08b4f0f9c6f3ecaf7850

Reply:
5a0c17c36d48a0fc1a44c392136dba2d64c6f6cd